presto-spi/src/main/java/com/facebook/presto/spi/type/Decimals.java

```diff
@@ -129,28 +127,12 @@ private static String getMatcherGroup(Matcher matcher, int group)
 
     public static Slice encodeUnscaledValue(BigInteger unscaledValue)
```

Comment:
I think we can remote such methods when we use UnscaledDecimalArithmetic in all code base. I would leave them for now and remove them at the end